### PR TITLE
chore: manually update license-check-and-add to 3.0.3 

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Description of changes

This PR manually updates license-check-and-add to 3.0.3 and makes the required changes for it to work. Thanks to @dbjorge who figured the rootcause and a proper way to handle the breaking changes from the version upgrade.

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@
 node_modules/
 .DS_STORE
 yarn-error.log
+.yarnrc
```
Comment (dbjorge) [inline .gitignore]: This shouldn't be .gitignored or deleted; it's intentional that it's checked in. See the comment in .yarnrc for the reasoning.

File: copyright-header.config.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+    "license": "copyright-header.txt",
+    "ignore": [
+        "**/*.svg",
```
Comment (dbjorge) [inline copyright-header.config.json]: Can you sort this list?

File: copyright-header.config.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "Dockerfile",
+        "**/*.snap",
+        "./.vscode",
+        "./.git",
+        "./.github",
+        "./dist",
+        "./drop",
+        "./extension",
+        "./node_modules",
+        "./copyright-header.txt",
```
Comment (dbjorge) [inline copyright-header.config.json]: Doesn't need to be here, it'll pass

File: copyright-header.config.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        "**/*.svg",
+        "**/*.scss.d.ts",
+        "**/.DS_Store",
+        "**/*.gitignore",
+        ".gitattribute",
+        "yarn.lock",
+        ".prettierignore",
+        ".yarnrc",
+        ".gitattributes",
+        "Dockerfile",
```
Comment (dbjorge) [inline copyright-header.config.json]: Dockerfile is interesting enough that it probably should be given a copyright header

File: copyright-header.config.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        "yaml|npmrc|yml|ps1": {
+            "eachLine": {
+                "prepend": "# "
+            }
+        },
+        "md": {
+            "prepend": "<!--",
+            "append": "-->"
+        },
+        "snap|ts|tsx|d.ts|js|scss|css": {
```
Comment (dbjorge) [inline copyright-header.config.json]: We can probably omit snap, since we're ignoring `**/*.snap` anyway

File: copyright-header.config.json (hunk 4 above)
Comment (Shobhit1) [reply copyright-header.config.json]: It has a copyright header but the checker cannot check for files without a type; I will have to check what it accepts for files which doesn't have a extension

File: copyright-header.config.json (hunk 3 above)
Comment (Shobhit1) [reply copyright-header.config.json]: Sure.

File: .gitignore (hunk 1 above)
Comment (Shobhit1) [reply .gitignore]: I missed that. Updated to restore it back.

File: copyright-header.config.json (hunk 2 above)
Comment (Shobhit1) [reply copyright-header.config.json]: Sure.

**Pull request merged**